### PR TITLE
Orders

### PR DESCRIPTION
**Pull request by caitlin-mohnike**

### CODE REVIEW

File: models/Order.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  Types = keystone.Field.Types;
+
+/**
+ * Order Model
+ * ==========
+ */
+
+var Order = new keystone.List('Order', {
+  map: { name: 'user' },
+  autokey: { path: 'slug', from: 'user', unique: true }
```
Comment (lizTheDeveloper) [inline models/Order.js]: Use the user's name and the release as the autokey's value creation scheme (the from attribute)

File: models/Order.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+});
+
+Order.add(  
+ 
+    {
+      	user: { type: Types.Relationship, ref: 'User', inital: true },
+      	dateToShip: { type: Types.Date },
+		dateOrderCreated: { type: Types.Date, default: Date.now },
```
Comment (lizTheDeveloper) [inline models/Order.js]: 'sup wit dese tabs

File: models/Order.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+var Order = new keystone.List('Order', {
+  map: { name: 'user' },
+  autokey: { path: 'slug', from: 'user', unique: true }
+});
+
+Order.add(  
+ 
+    {
+      	user: { type: Types.Relationship, ref: 'User', inital: true },
```
Comment (lizTheDeveloper) [inline models/Order.js]: initial not inital

File: models/Order.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      	dateToShip: { type: Types.Date },
+		dateOrderCreated: { type: Types.Date, default: Date.now },
+		orderPriceTotal: { type: Types.Money },
+		beerRelease: { type: Types.Relationship, ref: 'Releases' },
+		orderStatus: { type: Types.Select, options: ['Prepaid', 'Refunded', 'Cancelled', 'Shipped', 'Processing', 'Unreleased', 'New'], default: 'New' },
+		otherProducts: { type: Types.Select, options: ['T-shirt', 'Beer Mugs'] },
+		OtherProductsPriceTotal: { type: Types.Money },
+		individualBeers: {type: Types.Relationship, ref: 'beers' },
```
Comment (lizTheDeveloper) [inline models/Order.js]: this needs to be an array

File: models/Order.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+		individualBeersPriceTotal: { type: Types.Money },
+		isGift: {type: Types.Boolean }
+
```
Comment (lizTheDeveloper) [inline models/Order.js:31]: also need order notes, delivery notes, gift message

File: models/User.js [hunk 6]
```diff
@@ -51,6 +51,7 @@ User.schema.virtual('canAccessKeystone').get(function() {
 
 User.relationship({ ref: 'Post', path: 'posts', refPath: 'author' });
 User.relationship({ ref: 'Credit', path: 'credit', refPath: 'user' });
+User.relationship({ ref: 'Order', path: 'order', refPath: 'user' });
```
Comment (lizTheDeveloper) [inline models/User.js]: path: 'orders'

File: models/Order.js (hunk 2 above)
Comment (caitlin-mohnike) [inline models/Order.js]: Fixed, I think. :)

File: models/Order.js (hunk 3 above)
Comment (caitlin-mohnike) [inline models/Order.js]: Done.

File: models/Order.js (hunk 4 above)
Comment (caitlin-mohnike) [inline models/Order.js]: I can't tell which line this is referring to...

File: models/Order.js (hunk 5 above)
Comment (caitlin-mohnike) [inline models/Order.js:31]: Added all as textareas

File: models/User.js (hunk 6 above)
Comment (caitlin-mohnike) [inline models/User.js]: Why is path orders?

File: models/Order.js (hunk 4 above)
Comment (caitlin-mohnike) [inline models/Order.js]: fixed. Added many:true

File: models/User.js (hunk 6 above)
Comment (caitlin-mohnike) [inline models/User.js]: Fixed. Also changed path for credits.

File: models/Order.js (hunk 1 above)
Comment (caitlin-mohnike) [inline models/Order.js]: Done.

File: models/User.js [hunk 7]
```diff
@@ -7,14 +7,14 @@ var keystone = require('keystone'),
  */
 
 var User = new keystone.List('User', {
-  map: { name: 'userName' },
-  autokey: { path: 'slug', from: 'userName', unique: true }
+  map: { name: 'emailAddress' },
```
Comment (lizTheDeveloper) [inline models/User.js]: change this BACK

File: models/beers.js [hunk 8]
```diff
@@ -31,6 +31,7 @@ Beers.add(
 );
 
 Beers.relationship({ ref: 'Releases', path: 'Releases', refPath: 'beersOfRelease'});
+Beers.relationship({ ref: 'Order', path: 'order', refPath: 'individualBeers'});
```
Comment (lizTheDeveloper) [inline models/beers.js]: Is this orders that the individual beer is present in? If so, it's plural.

File: models/beers.js [hunk 9]
```diff
@@ -31,6 +31,7 @@ Beers.add(
 );
 
 Beers.relationship({ ref: 'Releases', path: 'Releases', refPath: 'beersOfRelease'});
+Beers.relationship({ ref: 'Orders', path: 'orders', refPath: 'individualBeers'});
```
Comment (lizTheDeveloper) [inline models/beers.js]: singular for models, plural for many relationships

File: models/Releases.js [hunk 10]
```diff
@@ -21,7 +21,7 @@ Releases.add(
 );
 
 Releases.relationship({ ref: 'User', path: 'User', refPath: 'Releases'});
-
+Releases.relationship({ ref: 'Order', path: 'order', refPath: 'beerRelease'});
```
Comment (lizTheDeveloper) [inline models/Releases.js]: this should also be plural (orders)

**Pull request merged**